torch/quantization/quantize.py

```diff
@@ -125,28 +125,24 @@ def add_quant_dequant(module):
         module._modules[name] = add_quant_dequant(child)
     return module
 
-def prepare(model, qconfig_dict=None):
-    r"""Prepares the model for calibration or training given a qconfig_dict.
+def prepare(model):
+    r"""Prepares the model for calibration or training.
```

Comment:
you probably want to expand comments on qconfig arguments being set on modules. Same comment for other functions below